pkg/manager/service_egress.go

```diff
@@ -180,6 +182,31 @@ func (sm *Manager) configureEgress(vipIP, podIP, namespace string, annotations m
 		protocol = iptables.ProtocolIPv6
 	}
 
+	// Use the internal egress implementation
+	if internalEgress != "" {
+		// Create an array of CIDRs that we wont SNAT to.
+		ignoreCIDRs := []string{
+			podCidr,
+			serviceCidr,
+		}
+
+		// Ad anny specifically denied networks
+		if deniedNetworks != "" {
+			networks := strings.Split(deniedNetworks, ",")
```

Comment:
dumb question: can deniedNetworks have space? is it work maybe trimming spaces?

Reply:
yeah, I think for safety reasons we potentially should do that..

`10.0.0.0/24, 192.168.0.0/24` etc would probably crash the function :|